src/Symfony/Bundle/TwigBundle/Resources/views/exception.css.twig

```diff
@@ -84,8 +84,8 @@ header .container { display: flex; justify-content: space-between; }
 .exception-message { flex-grow: 1; }
 .exception-message, .exception-message a { color: #FFF; font-size: 21px; font-weight: 400; margin: 0; }
 .exception-message.long { font-size: 18px; }
-.exception-message a { text-decoration: none; }
-.exception-message a:hover { text-decoration: underline; }
+.exception-message a { border-bottom: 1px solid rgba(255, 255, 255, 0.5); font-size: inherit; text-decoration: none; }
```

Comment:
Is really `text-decoration: underline` too "aggressive"?

Reply:
Well, NVM, it provides feedback on hover.

Reply:
Is not aggressive ... but it collides with underscores in template paths (which are super common in Twig templates) so it's a bit confusing.